main/src/ca/uwaterloo/flix/language/errors/WeederError.scala

```diff
@@ -679,6 +679,30 @@ object WeederError {
 
   }
 
+  /**
+    * An error raised to indicate that the case of an alias does not match the case of the original value.
+    *
+    * @param patt     the invalid regular expression
+    * @param loc      the location where the error occurred
+    */
+  case class InvalidRegularExpression(patt: String, loc: SourceLocation) extends IllegalLiteral {
+    def summary: String = s"The pattern literal '${patt}' is not a valid regular expression."
```

Comment:
Might it not be useful to include the underlying error message from Java? 

Otherwise it might be hard to understand what is broken.

Reply:
Done